src/test/java/seedu/address/model/property/PropertyTest.java

```diff
@@ -54,6 +58,21 @@ public void isSameProperty() {
     }
 
     @Test
+    @DisplayName("Test for getAskingPrice method, which checks for existence of client. ")
+    public void getAskingPrice() {
+        // no client
+        Property mayfair = new PropertyBuilder(MAYFAIR).withName(VALID_NAME_BURGHLEY_DRIVE).build();
```

Comment:
Is there a reason for using `.withName(VALID_NAME_BURGHLEY_DRIVE)`?

Reply:
For completeness I guess, maybe can remove.